smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/httpResponse/bindingTraits/XMLHttpResponseTraitResponseCode.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+package software.amazon.smithy.swift.codegen.integration.httpResponse.bindingTraits
+
+import software.amazon.smithy.model.traits.HttpResponseCodeTrait
+import software.amazon.smithy.swift.codegen.SwiftWriter
+import software.amazon.smithy.swift.codegen.integration.HttpBindingDescriptor
+import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
+
+class XMLHttpResponseTraitResponseCode(
+    val ctx: ProtocolGenerator.GenerationContext,
+    val responseBindings: List<HttpBindingDescriptor>,
+    val writer: SwiftWriter
+) {
+    fun render() {
```

Comment:
Renders statements to set output model values from HTTP response code.
Used to construct a HTTP binding for an output model.